src/scripts.js

```diff
@@ -297,12 +306,53 @@ function findRecipesWithCheckedIngredients(selected, ingredientsRepository) {
   })
 }
 
-function getRecipeCost() {
+const getRecipe = () => {
   let recipeId = event.path.find(e => e.id).id;
   let recipe = recipes.find(recipe => recipe.id === Number(recipeId));
+  if (event.target.classList.value === 'calculate-cost') {
+    getRecipeCost(recipe)
+  } else if(event.target.classList.value === 'check-pantry') {
+    checkPantryForIngredients(recipe)
+  } else if(event.target.classList.value === 'add-ingredients-btn') {
+    putItemsInPantry(recipe)
+  } else if(event.target.classList.value === 'cook-recipe') {
+    cookRecipe(recipe)
+  }
+}
+
+function getRecipeCost(recipe) {
   let userChecked = user.checkUserPantryForIngredients(recipe);
-  // let checkedPantry = user.pantry.checkPantryForIngredients(recipe.ingredients);
-  // let checkedAmount = user.pantry.checkIngredientAmount(recipe.ingredients);
-  console.log(userChecked);
-    
+  let costOfRecipe = Math.round(ingredientsRepository.getIngredientsCost(userChecked)) / 100
+  domUpdates.displayRecipeCost(costOfRecipe)
+}
+
+const checkPantryForIngredients = (recipe) => {
+  let userChecked = user.checkUserPantryForIngredients(recipe);
+  let ingredientsWithNames = userChecked && userChecked.map(checkedIngredient => ingredientsRepository.getIngredientName(checkedIngredient, 'id'))
+  domUpdates.displayNeededIngredients(ingredientsWithNames, userChecked, recipe)
+}
+
+
+const putItemsInPantry = (recipe) => {
+    let userChecked = user.checkUserPantryForIngredients(recipe);
+    const pantryBefore = user.pantry.ingredients.length
+    user.addItemsToPantry(userChecked)
+    pantryBefore < user.pantry.ingredients.length ? findPantryInfo('update') : domUpdates.cookMessage(recipe)
+}
+
+const cookRecipe = (recipe) => {
+  if (event.target.classList.value === 'cook-recipe') {
+    user.removeRecipeIngredients(recipe.ingredients)
+    findPantryInfo('update')
+  }
+}
+
+const fixRecipeData = () => {
```

Comment:
Since a lot of these functions are ES6, should we go through and convert the rest of our functions/methods to ES6 as well?